pkg/models/services/sechub/secretstores/ark_sechub_create_secretstore.go

```diff
@@ -0,0 +1,46 @@
+package secretstores
+
+type ArkSecHubCreateSecretStoreConnectionConfig struct {
+	ConnectionType string `json:"connection_type,omitempty" mapstructure:"connection_type,omitempty" flag:"connection-type" desc:"COMMON - AKV, GCP: If your Cloud Vault is not open to public access, choose 'CONNECTOR'. Valid Values: 'CONNECTOR','PUBLIC"`
```

Comment:
For anything that has choices in it, use choices like here:
https://github.com/cyberark/ark-sdk-golang/blob/0f54adeaa82820d52cbb3fef6678834b1dbd23d2/pkg/models/services/sia/access/ark_sia_install_connector.go#L5